packages/backend-test-utils/src/next/services/mockServices.ts

```diff
@@ -57,6 +58,44 @@ function simpleFactory<
   })) as (...options: TOptions) => ServiceFactory<TService, any>;
 }
 
+/** @public */
+export type ServiceMock<TService> = {
+  factory: ServiceFactory<TService>;
+} & {
+  [Key in keyof TService]: TService[Key] extends (
+    this: infer This,
+    ...args: infer Args
+  ) => infer Return
+    ? TService[Key] & jest.MockInstance<Return, Args, This>
+    : TService[Key];
+};
+
+/** @internal */
+function simpleMock<TService>(
+  ref: ServiceRef<TService, any>,
```

Comment:
```suggestion
  ref: ServiceRef<TService, unknown>,
```

i dunno, just don't like seeing `any` where not strictly necessary, especially in inputs :)

Reply:
It has to be `any`. With `unknown` it doesn't satisfy the constraint, and with `'root' | 'plugin'` it fails to detect any overload.